scripts/pi-hole/php/footer.php

```diff
@@ -65,10 +65,10 @@
         <div class="row row-centered text-center version-info">
             <div class="col-xs-12 col-sm-12 col-md-10">
                 <ul class="<?php echo $list_class; ?>">
-                    <?php if ($docker_current) { ?>
+                    <?php if ($dockerVersionStr) { ?>
```

Comment:
I used `$docker_current` here for consistency and because its' value is a _boolean_.

Reply:
`$docker_current` is a string... it's currently `2022.09`